src/codeu/chat/DerbyStore.java

```diff
@@ -122,6 +168,7 @@ public void addUser(User u) throws SQLException {
 		addUser.executeUpdate();
```

Comment:
Maybe some comments explaining exactly what these arguments are would be helpful. For example, is every username's password "test" right now? Is that by design, or a placeholder for something else in the future?